src/az-purview/deploy/terraform/modules/storage/main.tf

```diff
@@ -0,0 +1,89 @@
+terraform {
```

Comment:
There is an existing Azure Storage module at src/common/infrastructure/terraform/modules/private-storage-account/main.tf.  Is there an opportunity to leverage the existing one instead of creating a new module?

Reply:
Same as above, @promisinganuj any thoughts?

Reply:
Same as https://github.com/Azure-Samples/virtual-network-integration-recipes/pull/28#discussion_r1115806809

Reply:
I have raised an issue to refactor the terraform modules #30